### PR TITLE
Update .gitmodules

### PR DESCRIPTION
**Pull request by junghans**

### CODE REVIEW

**Comment by ychen-hpc** [comment]
It looks like the submodule link is still not clickable on GitHub
`* [`Sample geometry files`](MF-LBM-extFiles/geometry_files) (in submodule):`

**Comment by junghans** [comment]
Huh?
Screen Shot 2021-05-05 at 09 23 55

**Comment by junghans** [comment]
Ok I see, how about this?

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "MF-LBM-extFiles"]
 	path = MF-LBM-extFiles
-	url = https://github.com/lanl/MF-LBM-extFiles.git
```
Comment (ychen-hpc) [inline .gitmodules:3]: shall we move back to the absolute path? Does relative path work when someone forks the repo?

File: README.md [hunk 2]
```diff
@@ -70,11 +70,11 @@ MF-LBM [1] [2] is a high-performance lattice Boltzmann (LB) code for direct nume
 * sample geometry files, pre-processing and post-processing code included
   
 ### Components
-* [`Sample geometry files`](MF-LBM-extFiles/geometry_files) (in submodule):
-  * [`simple_geometry_tube_sphere`](MF-LBM-extFiles/geometry_files/tube_sphere_example/tube_sphere.dat): a tube with a spherical obstacle in the center  
-  * [`rock_sample_text_images`](MF-LBM-extFiles/geometry_files/sample_rock_geometry_textimage/bentheimer_oregon_state_240): text images from cropped Bentheimer sandstone scans [3]
-  * [`rock_sample_wall_array_converted`](MF-LBM-extFiles/geometry_files/sample_rock_geometry_wallarray/bentheimer_240_240_240.dat): binary file of the wall array converted from text images
-  * [`rock_sample_wall_array_processed`](MF-LBM-extFiles/geometry_files/sample_rock_geometry_wallarray/bentheimer_in10_240_240_240_out10.dat): binary file of the wall array converted from text images with added buffer layers 
+* [`Sample geometry files`](https://github.com/lanl/MF-LBM-extFiles) (in submodule):
```
Comment (ychen-hpc) [inline README.md:73]: This may be our best solution since the readme file focuses more on GitHub website.

File: .gitmodules (hunk 1 above)
Comment (junghans) [reply .gitmodules:3]: Relative paths are actually better, as it will use the same protocol everywhere, i.e. everything goes over ssh or https.

**Pull request merged**